pegasus/sites.v3/hourofcode.com/public/how-to/index.md.partial

```diff
@@ -6,77 +6,99 @@ nav: how_to_nav
 
-## 3. Promote your Hour of Code
-Promote your Hour of Code [with these tools]({{ urls/promote_resources }}) and encourage others to host their own events.
+We provide a variety of fun, [student-guided tutorials]({{ urls/learn }}) for all age groups and experience levels. It’s popular for students to try self-led tutorials, though many activities include lesson plans for teachers to guide discussion or extend the activity as well.
 
-## 4. Plan your technology needs - computers are optional
+Explore the activities and decide ahead of time if you want to choose a single tutorial for all of your students, or let each child pick their own.
 
-The best Hour of Code experience includes Internet-connected computers. But you **don’t** need a computer for every child, and you can even do the Hour of Code without a computer at all.
+[<img src="/images/fit-700/tutorials.png">]({{ urls/learn }})
 
-Make sure to test tutorials on student computers or devices to ensure they work properly on browsers with sound and video. **Have low bandwidth?** Plan to show videos at the front of the class, so each student isn't downloading their own videos. Or try the unplugged / offline tutorials.
+## 3. Create your Plan for the Day
 
-Provide headphones for your class, or ask students to bring their own, if the tutorial you choose works best with sound.
+### Think about your technology needs - computers are optional!
 
-**Don't have enough devices?** Use [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q). When students partner up, they help each other and rely less on the teacher. They’ll also see that computer science is social and collaborative.
+- The best Hour of Code experience includes Internet-connected computers. But you don’t need a computer for every child, and you can even do the Hour of Code without a computer at all! For [unplugged activities](/learn), simply filter the Classroom Technology section to show options for “No computers or devices”.
+- Make sure to test tutorials on student computers or devices to ensure they work properly on browsers with sound and video. Have low bandwidth? Plan to show videos at the front of the class, so each student isn't downloading their own videos. Or try the offline tutorials.
+- Provide headphones for your class, or ask students to bring their own, if the tutorial you choose works best with sound.
+- Don't have enough devices? Use [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q). When students partner up, they help each other and rely less on the teacher. They’ll also see that computer science is social and collaborative.
 
 <img src="/images/fit-350/group_ipad.jpg">
 
-## 5. Start your Hour of Code off with an inspiring speaker or video
+### Pick a day and time
+
+People around the world join in the Hour of Code celebration during CS Education Week (December 9-13) when the latest tutorials and activities are released. But you can do an Hour of Code any day of the year!
```

Comment:
Can we make these dates data-driven?  I know we have dates in pegasus/sites.v3/hourofcode.com/i18n/countries.json but not sure that we've ever used them on code.org.